src/Kliva/XAMLResources/DataTemplates.xaml

```diff
@@ -16,58 +17,77 @@
     <DataTemplate x:Key="ActivityListItemDataTemplate"
                   x:DataType="models:ActivitySummary">
         <!-- UserControl needed for visual state manager -->
-        <UserControl>
-          <RelativePanel x:Name="ActivityListItemPanel"
+        <uwpToolkitControls:SlidableListItem x:Name="SlidableActivityListItem"
+                                             HorizontalAlignment="Stretch"
+                                             MouseSlidingEnabled="True"
+                                             LeftBackground="{StaticResource KlivaMainBrush}"
+                                             LeftCommand="{Binding Path=Main.KudosCommand, Source={StaticResource Locator}}"
+                                             LeftCommandParameter="{x:Bind}"
+                                             LeftIcon="Like"
+                                             LeftLabel="Give Kudos"
+                                             RightBackground="{StaticResource KlivaMainBrush}"
+                                             RightCommand="{Binding Path=Main.CommentCommand, Source={StaticResource Locator}}"
+                                             RightCommandParameter="{x:Bind}"
+                                             RightIcon="Comment"
+                                             RightLabel="Comment">
+            <interactivity:Interaction.Behaviors>
+                <core:DataTriggerBehavior Binding="{x:Bind HasKudoed, Mode=OneWay}" ComparisonCondition="Equal" Value="True">
```

Comment:
This does not trigger 'after' given a Kudos. This only works when loading the list with items. Even setting it to TwoWay has no effect.
We need to see how we can fix this...